src/mfakto.cpp

```diff
@@ -152,6 +153,7 @@ int init_CLstreams(int gs_reinit_only)
       printf("ERROR: malloc(h_RES) failed\n");
       return 1;
     }
+    memset(mystuff.h_RES, 0, sizeof(mystuff.h_RES));
```

Comment:
This is using a pointer size to zeroize the data it points to. Modern gcc (14.2.1 in my case) flags it as a warning. This should probably be `memset(mystuff.h_RES, 0, sizeof(*mystuff.h_RES));`

Reply:
Thanks for this catch, it wasn't flagged as a warning with MSC 19.40.33811.

Reply:
I also got some warnings on macOS, but your latest changes resolved them.